restart.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Restart zuul services
+  hosts: zuul
+  become: yes
+  tags:
+    - zuul
+    - zuul-server
+    - zuul-launcher
+  tasks:
+    - name: restart zuul-server
+      service:
+        name: zuul-server
+        state: restarted
+
+    - name: wait for zuul to start (check for gearman)
+      wait_for:
+        port: "{{ zuul_gearman_port | default(4730) }}"
+        timeout: 300
+        delay: 2
+
+    - name: restart zuul-merger
```

Comment:
This should just restart zuul-launcher, we want to restart zuul-merger on "hosts: mergers"